### ISSUE TITLE
md-raised-button on anchor always have disabled styles 

### ISSUE THREAD

**Issue by fxck** [angular/material2#1946]
it appears to have `disabled="false"`, which matches these rules

```
[md-raised-button].md-primary[disabled], [md-raised-button].md-accent[disabled], [md-raised-button].md-warn[disabled], [md-raised-button][disabled][disabled], [md-fab].md-primary[disabled], [md-fab].md-accent[disabled], [md-fab].md-warn[disabled], [md-fab][disabled][disabled], [md-mini-fab].md-primary[disabled], [md-mini-fab].md-accent[disabled], [md-mini-fab].md-warn[disabled], [md-mini-fab][disabled][disabled]
```

it shouldn't have `disabled` attribute at all(should be `undefined` instead of `false`).

would make plunker, but it appears to be outdated( @DevVersion )

![image](https://cloud.githubusercontent.com/assets/1303561/20482167/6dae3c74-afeb-11e6-8c83-6904ff9a14ac.png)

**Comment by fxck** [comment]
apparently was fixed in https://github.com/angular/material2/commit/716372bc856a71f18c3d27185a0a8bfae82e41fd#diff-a9be5af2302f13e68145b436ee01073aR145 

?

**Comment by jelbourn** [comment]
Should be fixed already in master, will be released in alpha.11

**Comment by fxck** [comment]
https://github.com/angular/material2/milestone/12 is this accurate?

**Comment by jelbourn** [comment]
@fxck nope, I've been terrible about keeping the milestones up to date.

**Comment by fxck** [comment]
You don't say. 😄 OK, are there any big features blocking it, or just some polishing to do?

**Comment by jelbourn** [comment]
@fxck #1915, #1907, #2026, #2014, and a fix for #1973
